actor/todo/src/lib.rs

```diff
@@ -0,0 +1,276 @@
+use std::collections::HashMap;
+
+use anyhow::{anyhow, Result};
+use log::{info, trace, warn};
+use serde::{Deserialize, Serialize};
+use wasmbus_rpc::actor::prelude::*;
+use wasmcloud_interface_httpserver::{HttpRequest, HttpResponse, HttpServer, HttpServerReceiver};
+use wasmcloud_interface_keyvalue::{
+    IncrementRequest, KeyValue, KeyValueSender, SetAddRequest, SetDelRequest, SetRequest,
+};
+
+// FIXME: contribute this upstream
```

Comment:
I can make a second PR once https://github.com/wasmCloud/interfaces/pull/10 is merged and released, or we can leave this PR open until then (this won't block anything on our side).